### PR TITLE
add more detail to Permissioning tutorial

### PR DESCRIPTION
**Pull request by macfarla**
Signed-off-by: Sally MacFarlane <sally.macfarlane@consensys.net>

## Describe the change

Added specific instructions to the Permissioning Tutorial, including besu command line and environment variables.

## Issue fixed

see #156 

### For content changes

- [x ] Doc content
- [ ] Doc pages organisation

### CODE REVIEW

File: docs/Tutorials/Permissioning/Getting-Started-Onchain-Permissioning.md [hunk 1]
```diff
@@ -81,15 +81,29 @@ Create the following environment variables and set to the specified values:
...
+```
+
+## Onchain permissioning command line options
+
 !!! important
 
-    The specified node must be a miner (PoW networks) or validator (PoA networks).
+    The specified node must be producing blocks, that is, be a miner (PoW networks) or validator (PoA networks).
+
+    To allow MetaMask to connect, the node must have JSON-RPC HTTP enabled and have rpc-http-cors-origins set to allow MetaMask.
```
Comment (bgravenorst) [inline docs/Tutorials/Permissioning/Getting-Started-Onchain-Permissioning.md:102]:
```suggestion
    To allow MetaMask to connect, the node must have JSON-RPC HTTP enabled, and have `--rpc-http-cors-origins` set to allow MetaMask.
```

File: docs/Tutorials/Permissioning/Getting-Started-Onchain-Permissioning.md [hunk 2]
```diff
@@ -81,15 +81,29 @@ Create the following environment variables and set to the specified values:
...
+
+    A simple way to set multiple environment variables is to create a file called `.env` with the required settings
+
+```env
+NODE_INGRESS_CONTRACT_ADDRESS=0x0000000000000000000000000000000000009999
+ACCOUNT_INGRESS_CONTRACT_ADDRESS=0x0000000000000000000000000000000000008888
+BESU_NODE_PERM_ACCOUNT=627306090abaB3A6e1400e9345bC60c78a8BEf57
+BESU_NODE_PERM_KEY=c87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3
+CHAIN_ID=2018
+```
```
Comment (bgravenorst) [inline docs/Tutorials/Permissioning/Getting-Started-Onchain-Permissioning.md:94]: Might be best to shift this block 4 spaces across to include it in the tip.

File: docs/Tutorials/Permissioning/Getting-Started-Onchain-Permissioning.md (hunk 1 above)
Comment (macfarla) [reply docs/Tutorials/Permissioning/Getting-Started-Onchain-Permissioning.md]: done

File: docs/Tutorials/Permissioning/Getting-Started-Onchain-Permissioning.md (hunk 2 above)
Comment (macfarla) [reply docs/Tutorials/Permissioning/Getting-Started-Onchain-Permissioning.md]: done

**Pull request merged**